src/components/common/CopyButton/index.tsx

```diff
@@ -4,22 +4,24 @@ import CopyIcon from '@/public/images/common/copy.svg'
 import { IconButton, SvgIcon } from '@mui/material'
 import CopyTooltip from '../CopyTooltip'
 
-const CopyButton = ({
-  text,
-  className,
-  children,
-  initialToolTipText = 'Copy to clipboard',
-  onCopy,
-  dialogContent,
-}: {
+export interface ButtonProps {
```

Comment:
Is this something we have to do for Storybook and does it make a different if we use type or interface here?

Reply:
No, I was plaing with this tutorial for enabling MUI in the preview:
https://storybook.js.org/recipes/@mui/material
and this is a left over. 

Basically with code like this:
```
import { Button as MuiButton, ButtonProps as MuiButtonProps } from '@mui/material';

export interface ButtonProps extends MuiButtonProps {
  label: string;
}
```

our Props would be also extended with all the available MUI buttonProps. Then you get 100s of MUI props next to your own component props and can filter with Pick or Omit.
```
// Only include variant, size, and color
type ButtonBaseProps = Pick<MuiButtonProps, 'variant' | 'size' | 'color'>;
```